homeassistant/components/flo/entity.py

```diff
@@ -62,6 +62,11 @@ def should_poll(self) -> bool:
         """Poll state from device."""
         return False
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return {"device_id": self._device.id, "location_id": self._device.location_id}
```

Comment:
Why would we include device ID ?

Reply:
So they can be used in the services otherwise there is no way to discover them

Reply:
Will remove and convert to entity based service as outlined below